### PR TITLE
chore: initial release on NPM

### PR DESCRIPTION
**Pull request by TimoGlastra**
This sets up everything up for the release. 

I manually released the first version for now, as automatic releases require some additional setup with github actions and Hyperledger permissions.

@jakubkoci I added you as an admin on NPM 

the package is released as `aries-framework@0.0.1-beta-1`. 

`aries-framework-javascript` was already taken as an org name, so I went with `aries-framework`. Did you squat the name @jakubkoci?

https://npmjs.com/package/aries-framework

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,32 +52,37 @@ See the [Roadmap](https://github.com/hyperledger/aries-framework-javascript/issu
...
 
-> If you're having trouble running this project, please the the [troubleshooting](./TROUBLESHOOTING.md) section. It contains the most common errors that arise when first installing libindy.
+#### NodeJS
 
-### Installing
+Follow the instructions [here](https://github.com/hyperledger/indy-sdk/#installing-the-sdk) to install `libindy`. Also make sure to have the right tools installed for the [NodeJS wrapper](https://github.com/hyperledger/indy-sdk/tree/master/wrappers/nodejs#installing). The NodeJS wrapper link also contains some common troubleshooting steps.
+
+If you don't want to install the dependencies yourself, the [Dockerfile](./Dockerfile) contains everything needed to get started with the framework. See [Usage with Docker](#usage-with-docker) for more information.
+
+> If you're having trouble running this project, please the the [troubleshooting](./TROUBLESHOOTING.md) section. It contains the most common errors that arise when first installing libindy.
```
Comment (jakubkoci) [inline README.md:63]: Double "the" in "please the the [troubleshooting]"

**Review by jakubkoci — APPROVED** [review]
Awesome! Just a tiny typo.

**Comment by jakubkoci** [comment]
I'm not aware that I would register `aries-framework-javascript` and I double-checked that I don't have such a project in my npm profile 🤷‍♂️

File: README.md (hunk 1 above)
Comment (TimoGlastra) [reply README.md]: Updated!

**Comment by TimoGlastra** [comment]
> I'm not aware that I would register `aries-framework-javascript` and I double-checked that I don't have such a project in my npm profile 🤷‍♂️

Guess someone else had the idea of an aries framework for JavaScript. Well guess it made the `aries-framework-javascript` vs `aries-framework` easier.

**Pull request merged**